doc/man3/EVP_PKEY_fromdata.pod

```diff
@@ -50,9 +61,155 @@ EVP_PKEY_fromdata() return 1 for success and 0 or a negative value for
 failure.  In particular a return value of -2 indicates the operation is
 not supported by the public key algorithm.
 
+=head1 EXAMPLES
+
+These examples are very terse for the sake of staying on topic, which
+are the EVP_PKEY_fromdata() set of functions.  In real applications,
+BIGNUMs would be handled, and converted to byte arrays with
+BN_bn2nativepad(), but that's off topic here.
+
+=begin comment
+
+TODO Write a set of cookbook documents and link to them.
+
+=end comment
+
+=head2 Creating an RSA keypair using raw key data
+
+ #include <openssl/evp.h>
+
+ /*
+  * These are extremely small to make this example simple.  A real
+  * and secure application will not use such small numbers.  A real
+  * and secure application is expected to use BIGNUMs, and to build
+  * this array dynamically.
+  */
+ const unsigned long rsa_n = 0xbc747fc5;
+ const unsigned long rsa_e = 0x10001;
+ const unsigned long rsa_d = 0x7b133399;
+ const OSSL_PARAM[] = {
+     OSSL_PARAM_ulong("n", &rsa_n),
+     OSSL_PARAM_ulong("e", &rsa_e),
+     OSSL_PARAM_ulong("d", &rsa_d),
```

Comment:
That documentation is incorrect.  PR welcome!

Reply:
To clarify: the numbers are treated as BIGNUM on the receiving end, and we currently only support them as unsigned integers in the OSSL_PARAM structure.  This is a deliberate choice made very early on to avoid having to do what's done with an ASN1 INTEGER for positive numbers with the highest bit set, and after having concluded that all key numbers are always positive anyway.

Reply:
As far as I know, OSSL_PARAM doesn't do BIGNUM, the numbers we pass with OSSL_PARAM are in the native endian format. BIGNUM isn't  in the native endian format (on all platforms). OSSL_PARAM makes a difference between signed and unsigned integers.

It makes perfect sense for those parameters to only support unsigned integers, but the documentation currently says only signed integers are supported. If you don't want to fix that in this PR, please open an issue, I do not have the time to create PRs.

Reply:
Heh, we shouldn't compare our piles of PRs...  I think I win in backlog size.

But...  it's a fairly minimal change after all, so I guess I can include it here.

Reply:
Darn.  There are *many* incorrectly documented numbers.  OK, different PR coming up.

Reply:
See #11275